### PR TITLE
fix(GNB): refactor header and apply new design to site map

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [x] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA

image

### CODE REVIEW

**Review by skdud4659 — APPROVED** [review]
LGTM!

File: apps/web/src/common/modules/navigations/gnb/modules/GNBHeader.vue [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+});
+const workspaceStore = useWorkspaceStore();
+const workspaceStoreState = workspaceStore.$state;
+const router = useRouter();
+
+const state = reactive({
+    ciLogoImage: computed(() => config.get('DOMAIN_IMAGE.CI_LOGO')),
+    workspaceList: computed(() => [...workspaceStoreState.getters.workspaceList]),
+    selectedWorkspace: computed<WorkspaceModel|undefined>(() => workspaceStoreState.getters.currentWorkspace),
+    workspaceMenuList: computed(() => state.workspaceList.map((_workspace) => {
```
Comment (WANZARGEN) [inline apps/web/src/common/modules/navigations/gnb/modules/GNBHeader.vue:37]: nitpick: What about to declare types? :)

File: apps/web/src/common/modules/navigations/gnb/modules/GNBHeader.vue [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+        }
+        return {
+            name: _workspace.workspace_id,
+            label: _workspace.name,
+        };
+    })),
+    searchText: '',
+});
+
+const selectWorkspace = (workspaceId) => {
```
Comment (WANZARGEN) [inline apps/web/src/common/modules/navigations/gnb/modules/GNBHeader.vue:54]: nitpick: What about to declare types?

**Review by WANZARGEN — APPROVED** [review]
Beautiful design!
I added some nitpick comments!

**Comment by WANZARGEN** [comment]
It seems review action has failed. 🥲

**Pull request merged**